server/src/db/migrations/20230804152934_renameImageUrl.cjs

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
when naming a migration we should specify which table we are affecting - `renameSitesImageUrl` (you don't need to change, but consider in the future)